Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageAssetProducer.cpp

```diff
@@ -93,14 +93,24 @@ namespace ImageProcessingAtom
         RPI::StreamingImageAssetCreator builder;
         builder.Begin(imageAssetId);
 
+        uint32_t imageDepth = m_imageObject->GetDepth(0);
+        const bool isVolumeTexture = m_imageObject->HasImageFlags(EIF_Volumetexture) || (imageDepth > 1);
+        if (m_imageObject->HasImageFlags(EIF_Cubemap) && isVolumeTexture)
+        {
+            AZ_Assert(false, "An image can not be a cubemap and a volume texture at the same time!");
```

Comment:
Should this be an AZ_Error so it is visible outside of debug builds?

Reply:
Actually, looking at AZ_Assert and AZ_Error it looks like both should exist in a profile build, for some reason I thought asserts were not visible outside debug builds.